app/views/govuk_publishing_components/components/docs/document_list.yml

```
@@ -156,6 +156,27 @@ examples:
           public_updated_at: 2017-07-19 15:01:48
           document_type: 'Statistical data set'
         subtext: 'First published during the 1996 Conservative Government'
+  without_underline:
+    description: Where the component is used for search results, the current design does not include underlines on links.
```

I think that removing the underlines for this component is a reasonably finder-frontend-specific use case that's worth mentioning that next to the component variation. The wording is definitely a suggestion, but I hope get's that across.

```suggestion
    description: The current search design does not include underlines on links and has been tested without underlines. Other uses will require further user testing.
```